src/symbol/glyph_source.js

```diff
@@ -94,11 +110,44 @@ class GlyphSource {
             if (!remaining) callback(undefined, glyphs, fontstack);
         };
 
-        for (const r in missing) {
+        for (const r in missingRanges) {
             this.loadRange(fontstack, r, onRangeLoaded);
         }
     }
 
+    createTinySDF(fontFamily, fontWeight) {
+        return  new TinySDF(24, 3, 8, .25, fontFamily, fontWeight);
+    }
+
+    loadCJKGlyph(fontstack, glyphID) {
+        let tinySDF = this.tinySDFs[fontstack];
+        if (!tinySDF) {
+            let fontWeight = '400';
+            if (/bold/i.test(fontstack)) {
+                fontWeight = '900';
+            } else if (/medium/i.test(fontstack)) {
+                fontWeight = '500';
+            } else if (/light/i.test(fontstack)) {
+                fontWeight = '200';
+            }
```

Comment:
These checks are only done once per unique fontstack, so probably only a handful of times.